gravitee-apim-gateway/gravitee-apim-gateway-reactor/pom.xml

```diff
@@ -96,5 +96,12 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+
+        <dependency>
```

Comment:
I added this dependency into here to solve the issue with reactor-core not found exception but I'm not sure if this is the best place to add this dependency